exampleCourse/questions/demo/addHTMLtable/server.py

```diff
@@ -7,18 +7,19 @@ def generate(data):
     name_greek_list = ["$\\gamma$", "$\\mu$", "$\\sigma$", "$\\lambda$", "$\\alpha$"]
 
     # Generate complete html string
-    mytable = (
-        '<table style="width:30%"><tr><th> Name </th> <th> Age </th> <th> Id </th></tr>'
-    )
+    mytable = '<table style="width:30%"><thead><tr><th> Name </th> <th> Age </th> <th> Id </th></tr></thead><tbody>'
```

Comment:
Nit, I think you're missing a corresponding `</tbody>`. I won't block on this since HTML parsers should still do the right thing, but we should still add it for completeness.

Reply:
Fixed in 9a4fe4a9c27b3ad017d20c4cdcb0f8cc74e1d55f.